include/mbgl/ios/MGLAnnotationImage.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
```

Comment:
Grabbed this last night already: https://github.com/mapbox/mapbox-gl-native/commit/56fbebf